app/src/main/java/com/example/home/login/LoginFragment.java

```diff
@@ -41,14 +44,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-        tvTitle = view.findViewById(R.id.tvTitle);
-        tvSubtitle = view.findViewById(R.id.tvSubtitle);
-        btnLogin = view.findViewById(R.id.btnLogin);
-        etEmail = view.findViewById(R.id.etEmail);
-        etPassword = view.findViewById(R.id.etPassword);
-        ivArrow = view.findViewById(R.id.ivArrow);
-        tvSwipeRight = view.findViewById(R.id.tvSwipeRight);
-        btnLogin.setOnClickListener(new View.OnClickListener() {
+        mtvTitle = view.findViewById(R.id.mtvTitle);
```

Comment:
the id does not need to have the "m" prefix. The m Prefix is only for Global Variables. However, for the IDs I'd recommend being from explicit (tvTitle -> TitleTextView)